concert/directors/scanning.py

```diff
@@ -9,12 +9,12 @@ class XYScan(Director):
     """
     Director to scan a specimen within a plane.
     """
-    x_min = Quantity(q.mm, check=['standby', 'error'])
-    x_max = Quantity(q.mm, check=['standby', 'error'])
-    x_step = Quantity(q.mm, check=['standby', 'error'])
-    y_min = Quantity(q.mm, check=['standby', 'error'])
-    y_max = Quantity(q.mm, check=['standby', 'error'])
-    y_step = Quantity(q.mm, check=['standby', 'error'])
+    x_min = Quantity(q.mm)
+    x_max = Quantity(q.mm)
+    x_step = Quantity(q.mm)
+    y_min = Quantity(q.mm)
+    y_max = Quantity(q.mm)
+    y_step = Quantity(q.mm)
```

Comment:
Why are the checks gone?

Reply:
I didn't check in detail, but I had the impression, that the check parameter requires a function and not a list of states. I will have a second look.